tests/manage/pv_services/test_rwo_pvc_fencing_unfencing.py

```diff
@@ -809,27 +796,31 @@ def test_rwo_pvc_fencing_node_prolonged_network_failure(
                 sleep=30,
             ), (f"App pod with name {pod_obj.name} did not reach Running state")
 
-        # Wait for mon and osd pods to reach Running state
-        selectors_to_check = [constants.MON_APP_LABEL, constants.OSD_APP_LABEL]
-        for selector in selectors_to_check:
-            assert ceph_cluster.POD.wait_for_resource(
-                condition=constants.STATUS_RUNNING,
-                selector=selector,
-                resource_count=3,
-                timeout=1800,
-                sleep=60,
-            ), f"3 expected pods with selector {selector} are not in Running state"
+        if not external_mode:
+            # Wait for mon and osd pods to reach Running state
+            selectors_to_check = {
+                constants.MON_APP_LABEL: 3,
```

Comment:
Maybe instead of the number "3", we can replace it with a var or constant `expected_mon_count`.

Reply:
done